daphne/src/messages/mod.rs

```diff
@@ -290,22 +360,33 @@ pub struct AggregateInitializeReq {
     pub report_shares: Vec<ReportShare>,
 }
 
-impl Encode for AggregateInitializeReq {
-    fn encode(&self, bytes: &mut Vec<u8>) {
+impl ParameterizedEncode<DapVersion> for AggregateInitializeReq {
+    fn encode_with_param(&self, encoding_parameter: &DapVersion, bytes: &mut Vec<u8>) {
```

Comment:
Explicit names are better (here and below).
```suggestion
    fn encode_with_param(&self, version: &DapVersion, bytes: &mut Vec<u8>) {
```